src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.html

```diff
@@ -1,4 +1,147 @@
-<exui-hearing-actuals-summary
-  [template]="template"
-  [mode]="mode">
-</exui-hearing-actuals-summary>
+<a class="govuk-back-link" href="javascript:history.back()" (click)="onBack()">Back</a>
+<main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
+  <div class="govuk-grid-row" *ngIf="hearingActualsMainModel">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l"></span>
+      <h1 class="govuk-heading-l">Hearing details</h1>
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Hearing date</dt>
+          <dd class="govuk-summary-list__value">{{hearingActualsMainModel.hearingActuals.actualHearingDays[0].hearingDate | date:"dd MMMM yyyy"}}</dd>
+        </div>
+      </dl>
+      <br>
+      <h1 class="govuk-heading-m">Hearing stage and result</h1>
+      <span class="govuk-body align-right">
+        <a href="javascript:void(0)" [routerLink]="['/', 'hearing-actual', 'stage-and-result']" class="govuk-link">Update</a>
+      </span>
+      <dl class="govuk-summary-list">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Hearing stage</dt>
+          <dd class="govuk-summary-list__value">{{hearingActualsMainModel.hearingActuals.hearingOutcome.hearingType}}</dd>
```

Comment:
if this hearingActualsMainModel.hearingActuals.hearingOutcome can put in the varialbe in tpyescript to get this shorter in the tempalte.